<?xml version="1.0"?>

<launch>

  <arg name="sensor_id" default="accelerometer_1" />
  <arg name="sensor_number" default="1" />

  <node pkg="accelerometer_ros" type="accelerometer_for_fusion" respawn="false" name="accelerometer_process" output="screen">
    <param name="sensor_id" type="string" value="$(arg sensor_id)"/>
    <param name="sensor_number" type="int" value="$(arg sensor_number)"/>
  </node>
</launch>
